deploy/002_deploy_optimism_adapter.ts

```diff
@@ -1,15 +1,16 @@
-import { L1_ADDRESS_MAP, USDC, WETH } from "./consts";
 import { DeployFunction } from "hardhat-deploy/types";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { CHAIN_IDs } from "../utils";
+import { L1_ADDRESS_MAP, OP_STACK_ADDRESS_MAP, USDC, WETH } from "./consts";
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { deployer } = await hre.getNamedAccounts();
   const chainId = parseInt(await hre.getChainId());
 
   const args = [
     WETH[chainId],
-    L1_ADDRESS_MAP[chainId].optimismCrossDomainMessenger,
-    L1_ADDRESS_MAP[chainId].optimismStandardBridge,
+    OP_STACK_ADDRESS_MAP[chainId][CHAIN_IDs.OPTIMISM].L1CrossDomainMessenger,
```

Comment:
To reduce chance of error, can we do something like `const addressMap = OP_STACK_ADDRESS_MAP[chainId][CHAIN_IDs.OPTIMISM]` above this line?  I worry we accidentally type in the wrong chain ID here and use the wrong address, which could be hard to detect

Same ask for all the deploy files, of course

Reply:
Yep; that's a good call.

In terms of catching errors, I think it'd be standard practice to test any deployed adapter with the chain adapter test task that was added in this commit: 8cc0bb59a4d70c0a69185f981a61964cd50d15c5.

With this we can verify that that tokens are relayed correctly. Bennett separately added a task to test messaging.